templates/rhpam78-authoring-ha.yaml

```diff
@@ -1219,6 +1219,10 @@ objects:
             value: "insecure-${APPLICATION_NAME}-kieserver"
           - name: KIE_SERVER_STARTUP_STRATEGY
             value: "OpenShiftStartupStrategy"
+          - name: KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED
+            value: "true"
+          - name: KIE_SERVER_CONTROLLER_OPENSHIFT_GLOBAL_DISCOVERY_ENABLED
```

Comment:
@desmax74 No need to change '..GLOBAL_DISCOVERY..'  as it's end user option.

Reply:
Ok @rhtevan